### ISSUE TITLE
Retain corner button functionality

### ISSUE THREAD

**Issue by brwcat** [torchgm/RoundedTB#43]
I've learned just how heavily etched into my brain it is to throw my cursor with the might of Zeus into the bottom left (and occasionally right) corner in order to open the Start menu (or show desktop, again on rare occasion). While more deliberate actions such as changing or opening apps is fine being more of a "floating" button, Start and Show Desktop tend to be more imprecise and speed-over-elegance type motions. 

Making the click area of Start and Show Desktop continue to extend to the bottom corner of the display would be a great improvement, at least as an option (sometimes people might not want this, i.e. floating taskbar or just not wanting the visual inconsistency).

**Comment by torchgm** [comment]
Unfortunately this is one of the things I chose to sacrifice - the way I'm cropping the taskbar does remove its hitbox, which is what drives the split/dynamic taskbar as well - the fact it's clickthroughable is one of the features I personally wanted to make sure was a thing. Re-adding the hitbox behind the taskbar and having its functionality map perfectly would be a really difficult task, so for now it's out-of-scope. ~~There are some future planned features in the works that might make it easier but shhhhhhh~~

However, R3 will allow you to customise the margins for each side of the taskbar independently so you could do something like one of these: 

![image](https://user-images.githubusercontent.com/31840547/134392670-2e4881e0-c5a6-49d8-9eef-1b7b1672dc08.png)
![image](https://user-images.githubusercontent.com/31840547/134392838-f67cf3a8-79ca-49e4-b4ea-4e01571ce4c2.png)

This will keep that hotcorner accessible whilst still being rather visually interesting.

**Comment by github-account1111** [comment]
Agreed. It *is* pretty frustrating having to aim for the Start icon when normally I could just throw the cursor there without even looking.

Pretty big sacrifice imo.

**Comment by b-** [comment]
Just wanted to comment in support of this. I love the look of RoundedTB, but unfortunately I can't justify using it unless there's a way to keep my hot corners.

**Comment by torchgm** [comment]
Oh I entirely forgot to mention I do have a solution for this now and I'll get around to implementing it for the next major release.

**Comment by b-** [comment]
This is exciting! Thanks for your hard work!

I want to mention as well that I'd love to have the aesthetics of a centered taskbar, the reason I have it left-aligned in Windows 11 is because of this corner hitbox functionality. If there's a way RoundedTB can make the corner hot even if the taskbar is centered, that would be the coolest thing!

**Comment by wynterthecat** [comment]
Another good Win11 feature to consider is splitting the Start, Search and etc buttons to the left while moving other buttons to the center.

**Comment by b-** [comment]
> Another good Win11 feature to consider is splitting the Start, Search and etc buttons to the left while moving other buttons to the center. 

Ah yes, this could even help balance the systray if you keep that visible
